nvi-evaluator/src/main/resources/sparql/nvi.sparql

```diff
@@ -0,0 +1,62 @@
+#Constructs nviType as either NviCandidate or NonNviCandidate based on the publication status,
+#publicationDate, publicationInstance and publicationContext levels.
+
+PREFIX  nva: <https://nva.sikt.no/ontology/publication#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+
+ASK
+WHERE {
+
```

Comment:
Since we know what we're selecting and not constructing anything (needs tesitng):

````
#Constructs nviType as either NviCandidate or NonNviCandidate based on the publication status,
#publicationDate, publicationInstance and publicationContext levels.

PREFIX  nva: <https://nva.sikt.no/ontology/publication#>
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>

ASK WHERE {
  VALUES ?approvedType { nva:AcademicMonograph nva:AcademicChapter nva:AcademicArticle nva:AcademicLiteratureReview }

  ?publication a nva:Publication ;
               nva:status nva:PUBLISHED ;
               (<>|!<>)+/nva:publicationInstance/rdf:type ?approvedType .


  ?publicationDate a nva:PublicationDate ;
                   nva:year "__NVI_YEAR__" .

  VALUES ?channelType { nva:Publisher  nva:Series   nva:Journal }
  VALUES ?level {"1" "2"}
  ?a a ?channelType ;
     nva:level ?level
}
```